features/step_definitions/us55-borrower-verified-landing-page.rb

```diff
@@ -0,0 +1,11 @@
+Given(/^I reach the Borrower Identity Verified Page$/) do
+  visit(Env.borrower_frontend)
+end
+
+Then(/^I should see "([^"]*)"$/) do |arg1|
+  page.should have_selector(:link_or_button, arg1)
+end
```

Comment:
Recommend using named variables rather than 'arg' to make it more readable/understandable.

There is no specific example of behaviour, its very vague, the person could be seeing a pink elephant for all we know.